src/Opdex.Platform.WebApi/Controllers/WalletsController.cs

```diff
@@ -100,9 +102,9 @@ public WalletsController(IMapper mapper, IMediator mediator)
     }
 
     /// <summary>Refresh Balance</summary>
-    /// <remarks>Retrieves and indexes the latest wallet public key balance for a token.</remarks>
-    /// <param name="address">The address of the wallet.</param>
-    /// <param name="token">The token to get the balance of.</param>
+    /// <remarks>Retrieves and indexes the latest balance of a token for an address.</remarks>
+    /// <param name="address">Address of the wallet.</param>
+    /// <param name="token">Address of the token to refresh the balance of, or CRS.</param>
```

Comment:
Think we can drop CRS support from this endpoint and return 400

Reply:
Sorted